lib/brakeman/parsers/template_parser.rb

```diff
@@ -39,6 +39,8 @@ def parse_template path, text
         tracker.error e, "could not parse #{path}"
```

Comment:
Should the same apply here?? ie `Could not parse...` ??

Reply:
Yes, got it in #1137